desktop/core/ext-py3/python-sasl-0.3.1/setup.py

```diff
@@ -38,13 +39,12 @@
             os.environ['MACOSX_DEPLOYMENT_TARGET'] = '10.9'
 
 
-sasl_module = Extension('sasl.saslwrapper',
-                        sources=['sasl/saslwrapper.cpp'],
+sasl_module = Extension('sasl.saslwrapper', ['sasl/saslwrapper.pyx'],
                         include_dirs=["sasl"],
                         libraries=["sasl2"],
                         language="c++")
 setup(name='sasl',
-      version='0.3.1',
```

Comment:
since this is not the full version upgrade, I think the way to say the version is the base version + patch SHA.
At least ourselves not confused by the version number.

Reply:
@wing2fly Hmm, good point. I have changed this as mention by you.